utils/bin/get-shards-clusterid

```diff
@@ -56,19 +56,32 @@ cmd.append(
 )
 res = subprocess.run(cmd, capture_output=True)
 described_clusters_raw = convert_to_dict(res)
-described_clusters_count = len(described_clusters_raw["items"])
-cluster_id = described_clusters_raw["items"][0]["id"]
+described_clusters_items = described_clusters_raw.get("items")
+if not described_clusters_items:
+    sys.stderr.write(
+        f"failed to parse response for {cmd}.\n verify that {cluster} is the right value you want or the ocm-cli is not configured correctly\n"
+    )
+    sys.exit(1)
 
+described_clusters_count = len(described_clusters_items)
```

Comment:
Nit: This is probably an unnecessary variable assignment.  Only used once, immediately after, and is 5 characters shorter.